docs/aws-provider.md

```diff
@@ -18,21 +10,51 @@ Note: You can only deploy into a VPC if the instance can be accessed by the mach
+1. Create an SSH key pair so that you can SSH into Bosh Lite once it is deployed. If you generate an EC2 Key Pair in AWS the private key will be downloaded. Call the EC2 Key Pair `bosh` or set the environment variable `BOSH_LITE_KEYPAIR` to the name you gave. Set `BOSH_LITE_PRIVATE_KEY` to the local file path for the private key (defaults to `~/.ssh/id_rsa_bosh`). 
 
-    |Name|Description|Default|
-    |---|---|---|
-    |BOSH_AWS_ACCESS_KEY_ID         |AWS access key ID                    | |
-    |BOSH_AWS_SECRET_ACCESS_KEY     |AWS secret access key                | |
-    |BOSH_LITE_KEYPAIR              |AWS keypair name                     |bosh|
-    |BOSH_LITE_NAME                 |AWS instance name                    |Vagrant|
-    |BOSH_LITE_SECURITY_GROUP       |AWS security group                   |inception|
-    |BOSH_LITE_PRIVATE_KEY          |path to private key matching keypair |~/.ssh/id_rsa_bosh|
-    |[VPC only] BOSH_LITE_SUBNET_ID |AWS VPC subnet ID                    | |
+To configure these environment variables all at once, copy and paste the following script into your shell then edit the file `aws-boshlite` to add values. 
+```
+cat > aws-boshlite <<EOF
+export BOSH_AWS_ACCESS_KEY_ID=
+export BOSH_AWS_SECRET_ACCESS_KEY=
+export BOSH_LITE_PRIVATE_KEY=
+EOF
+```
+Now set the environment variables by sourcing the file:
+```
+source aws-boshlite
+```
+### Additional Prerequisites for EC2 Classic
 
-    Note: `BOSH_LITE_SECURITY_GROUP` should be set to group id not the group name if VM is deployed into the VPC, e.g. `sg-11764446`
+1. Create a Security Group with name `inception`, or set the environment variable `BOSH_LITE_SECURITY_GROUP` to the Group Name of the security group you created. Do not use Group ID, as the deploy will fail unless the Security Group is associated with a VPC.
+1. Continue to [Deploy BOSH Lite](#deploy-bosh-lite).
 
-1. Make sure the security group you are using exists and allows inbound TCP traffic on ports 25555 (for the BOSH director), 22 (for SSH), 80/443 (for Cloud Controller), and 4443 (for Loggregator).
+### Additional Prerequisites for VPC
+
+1. If you don't already have one, create a VPC. If you use the VPC Wizard, a Security Group and a Subnet will be created for you. If you create the VPC manually a Security Group will be created automatically but you must manually create a Subnet.
+1. Set the environment variable `BOSH_LITE_SECURITY_GROUP` to the Group ID (e.g. `sg-62166d1a`) of a Security Group associated with the VPC. Note: this is different from [EC2-Classic](#additional-prerequisites-for-ec2-classic), where the Group Name is used.
+1. By default Security Groups only allow access from within the Security Group. Modify the Security Group to allow inbound traffic from anywhere (set Source to `0.0.0.0/0`). 
+  - If you want to lock down access, BOSH Lite requires inbound traffic on ports 25555 (for the BOSH director), 22 (for SSH), 80/443 (for Cloud Controller), and 4443 (for Loggregator).
```

Comment:
I added a commit to enhance the sub-bullet to say you can lock it down by setting the source to your IP and limit open ports to those required.